src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -207,12 +227,33 @@ export class RelayFeeCalculator {
 
     if (toBN(amountToRelay).eq(0)) return MAX_BIG_INT;
 
-    const getGasCosts = this.queries.getGasCosts(deposit, amountToRelay, relayerAddress).catch((error) => {
-      this.logger.error({ at: "sdk-v2/gasFeePercent", message: "Error while fetching gas costs", error });
-      throw error;
-    });
+    const getGasCosts = this.queries
+      .getGasCosts(
```

Comment:
Sorry, one last question: did you change this simulation to using nonzero fees (1 wei should be fine), just to match the previous behavior?

Reply:
It matches the deposit, but the API does pass in a non-zero fee. It's visible in the PR on the FE.

Reply:
Gotcha, okay, cool.